src/Di/Di.php

```diff
@@ -173,15 +176,13 @@ protected static function instantiable(string $type): bool
      */
     private static function userDependencies(): array
     {
-        $fs = new FileSystem();
-
         $userDependencies = base_dir() . DS . 'shared' . DS . 'config' . DS . 'dependencies.php';
 
-        if (!$fs->exists($userDependencies)) {
+        if (!file_exists($userDependencies)) {
```

Comment:
Qunatum's filesystem functionality could've been used here too.

Reply:
`Di` is low level class and can't have dependencies